proto/public/api.proto

```diff
@@ -182,6 +186,9 @@ message StatTable {
       uint64 failed_pod_count = 6;
 
       BasicStats stats = 5;
+
+      // propagate pod errors so we can display them to the user
```

Comment:
This would be slightly clearer to me with a comment like:

```
// Stores a set of errors for each pod name. If a pod has no errors, it may be omitted.
```

I.e. it's primarily important for me to know the semantics of this field.  How it's used is secondary (and, in my opinion, less important from the API docs perspective).

Reply:
agreed.